context/configSchema.js

```diff
@@ -1,153 +1,121 @@
-const Schema = require('validate');
+const Validator = require('fastest-validator');
 
-module.exports = new Schema({
+const validator = new Validator();
+
+module.exports = validator.compile({
   clientEntry: {
-    type: String,
-    required: true
+    type: 'string'
   },
   renderEntry: {
-    type: String,
-    required: true
+    type: 'string'
   },
   serverEntry: {
-    type: String,
-    required: true
+    type: 'string'
   },
   libraryEntry: {
-    type: String,
-    required: false
+    type: 'string',
+    optional: true
   },
   routes: {
-    type: Array,
-    each: {
-      name: {
-        type: String,
-        required: true
-      },
-      route: {
-        type: String,
-        required: true
-      },
-      entry: {
-        type: String,
-        required: false
+    type: 'array',
+    items: {
+      type: 'object',
+      props: {
+        route: {
+          type: 'string'
+        },
+        name: {
+          type: 'string',
+          optional: true
```

Comment:
Is it a deliberate change to make `name` optional? Previously `entry` was the only optional key.

Reply:
Yes, this was deliberate. It's technically not required and probably shouldn't have been in the first place.